Directory.Build.props

```diff
@@ -42,6 +42,11 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(Tooling_MicrosoftCodeAnalysisAnalyzersPackageVersion)" NoWarn="NU1608" />
     <PackageReference Include="Microsoft.NetCore.Analyzers" Version="$(Tooling_MicrosoftNetCoreAnalyzersPackageVersion)" NoWarn="NU1608" />
+    <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="$(Tooling_MicrosoftCodeAnalysisBannedApiAnalyzersPackageVersion)" />
```

Comment:
Super interesting, had no idea this was a thing. /cc @dougbu in case you all wanted to utilize something like this in the dotnet/aspnetcore side of the world.